### DISCUSSION TITLE
how to rotate a body smoothly?

### DISCUSSION THREAD

**Discussion by AmirHo3einJS** [pmndrs/p2-es#17] [Q&A] [answered]
Hello, So I want to set the angle of a body smoothly. for instance from 0 to 90 degrees. how can I do that? I wanted to use angular velocity or apply torque to the body but there is no ensure that the final angle of the body would be 90 degrees.

**Comment by isaac-mason** [comment, marked as answer]
The `control.html` demo contains a good example of smoothly rotating a body towards a target.

https://p2-es.pmnd.rs/demos/control.html
https://github.com/pmndrs/p2-es/blob/master/apps/p2-es-demos/src/control.html
